packages/lesswrong/components/spotlights/DismissibleSpotlightItem.tsx

```diff
@@ -6,20 +6,32 @@ import { Components, registerComponent } from '../../lib/vulcan-lib';
 import { useCookiesWithConsent } from '../hooks/useCookiesWithConsent';
 import { HIDE_SPOTLIGHT_ITEM_PREFIX } from '../../lib/cookies/cookies';
 
-export const CurrentSpotlightItem = ({classes}: {
+export const DismissibleSpotlightItem = ({
```

Comment:
It's a bit confusing that you can't bring it back once you have dismissed it. I think it would be better (and consistent with the topic page) if it just wasn't dismissible. For the frontpage spotlight it makes sense for it to be dismissible because it changes plus it takes up valuable space on the frontpage, but on this page I feel space isn't as valuable